include/AMR.h

```diff
@@ -152,6 +157,15 @@ struct AMR_t
          PotSgTime[lv][   PotSg[lv] ] = -__FLT_MAX__;
          PotSgTime[lv][ 1-PotSg[lv] ] = -__FLT_MAX__;
 #        endif
+
+//       use fluid solver by default for hybrid scheme
+#        if ( ELBDM_SCHEME == ELBDM_WAVE )
+         use_wave_flag[lv] = true;
+#        elif ( ELBDM_SCHEME == ELBDM_HYBRID )
+         use_wave_flag[lv] = false;
+#        else
+#        error : UNKNOWN ELBDM_SCHEME !!
+#        endif // # if ELBDM_SCHEME
```

Comment:
This part needs to be inside the `ELBDM` only region:
`#  if ( MODEL == ELBDM )`
`#  endif // #if ( MODEL == ELBDM )`

Reply:
@JeiwHuang 
Thanks for pointing that out.